src/nvimage/index.ts

```diff
@@ -3560,7 +3560,7 @@ export class NVImage {
    */
   toUint8Array(drawingBytes: Uint8Array | null = null): Uint8Array {
     const isDrawing = drawingBytes !== null
-    const hdrBytes = hdrToArrayBuffer(this.hdr!, isDrawing)
+    const hdrBytes = hdrToArrayBuffer({ ...this.hdr!, vox_offset: 352 }, isDrawing)
```

Comment:
@cdrake and @neurolabusc , by forcing `vox_offset: 352`, is there a possibility of saving a corrupted nifti file if the original file had extensions present? `toUint8Array` could be called on any input image provided to NiiVue, not just images we create with internal functions.

Reply:
@hanayik no, when we save an image to NIfTI we remove any extensions added to image, and the image is always at offset 352. This is intentional, as we do only want to retain data we can interpret. @cdrake noted this change is required. Our sample `aal.nii.gz` exhibits this - the header offset is not 352 since it includes atlas labels as text inserted after the binary header and before the image.

Reply:
@hanayik - No, because the toUint8Array method for NVImage (which is what NVDocument uses to serialize volumes) sets the header as 348 bytes (through the hdrToArrayBuffer method), adds 4 bytes and then adds the img array at the end. It does mean we do not preserve extensions using this method.

``` typescript
toUint8Array(drawingBytes: Uint8Array | null = null): Uint8Array {
    const isDrawing = drawingBytes !== null
    const hdrBytes = hdrToArrayBuffer({ ...this.hdr!, vox_offset: 352 }, isDrawing)
    ...
    const img8 = isDrawing ? (drawingBytesToBeConverted as Uint8Array) : new Uint8Array(this.img!.buffer)
    const opad = new Uint8Array(4)
    const odata = new Uint8Array(hdrBytes.length + opad.length + img8.length)
    odata.set(hdrBytes)
    odata.set(opad, hdrBytes.length)
    odata.set(img8, hdrBytes.length + opad.length)
    return odata
  }
```
  
``` typescript
export function hdrToArrayBuffer(hdr: NiftiHeader, isDrawing8 = false): Uint8Array {
  const SHORT_SIZE = 2
  const FLOAT32_SIZE = 4

  const byteArray = new Uint8Array(348)
```